libtcmu_log.c

```diff
@@ -481,6 +482,7 @@ int tcmu_create_file_output(int pri, const char *filename, bool reloading)
 		return ret;
 	}
 
+	tcmu_info("log file path now is \"%s\"\n", log_file_path);
```

Comment:
I think we should move this to the end of the function in case we fail.

If you want a message above I think it makes more sense as a debug message and something like

tcmu_dbg("Attempting to use %s as the log file path\n", log_file_path);

then at the end of the function when we are successful do:

tcmu_info("log file path is now \"%s\"\n", log_file_path);

Reply:
@pkalever Looks good.

@mikechristie Yeah, make sense.